frontend/ui/widget/textboxwidget.lua

```diff
@@ -304,10 +305,10 @@ function TextBoxWidget:onHoldWord(callback, ges)
                 -- now find which word the character is in
                 local words = util.splitToWords(line.text)
                 local probe_idx = char_start
-                for _,w in ipairs(words) do
+                for _, w in ipairs(words) do
                     -- +1 for word separtor
-                    probe_idx = probe_idx + string.len(w)
```

Comment:
For multi-byte unicode characters it will cause index mismatch, as the rest part are all using character index.